packages/expo-updates/ios/EXUpdates/CodeSigning/EXUpdatesCertificateChain.swift

```diff
@@ -14,6 +14,10 @@ public class EXUpdatesExpoProjectInformation : NSObject {
     self.projectId = projectId
     self.scopeKey = scopeKey
   }
+  
+  public static func ==(lhs: EXUpdatesExpoProjectInformation, rhs: EXUpdatesExpoProjectInformation) -> Bool {
```

Comment:
Make the class also conform to `Equatable` in the class declaration

Reply:
`NSObject` already conforms to the `Equatable` protocol (though that's new as of swift 2).